packages/client/src/artifact/checkout.rs

```diff
@@ -7,8 +7,12 @@ use std::{path::PathBuf, pin::pin};
 use tangram_futures::stream::TryStreamExt as _;
 use tangram_http::{incoming::response::Ext as _, outgoing::request::Ext as _};
 
-#[derive(Clone, Debug, Default, serde::Deserialize, serde::Serialize)]
+#[derive(Clone, Debug, serde::Deserialize, serde::Serialize)]
 pub struct Arg {
+	/// Whether to check out the artifact's dependencies.	#[arg(long)]
```

Comment:
Remove the stray clap attribute in this comment.

Reply:
done